autoPlay.user.js

```diff
@@ -248,6 +250,9 @@ function disableParticles() {
 }
 
 function MainLoop() {
+	var cHours = cTime.getUTCHours();
```

Comment:
Ops... Both cHours and cMins must be globals for the useWormholeIfRelevant to access.
Fixing...